engines/griffon/resources.cpp

```diff
@@ -293,26 +293,36 @@ void GriffonEngine::loadMap(int mapnum) {
 					rcDest.setWidth(16);
 					rcDest.setHeight(16);
 
+					int flag=0;
```

Comment:
The line does not conform to our code formatting guidelines. It must be `int flag = 0;` Note spaces around the operator.